azure-iot-device/azure/iot/device/provisioning/aio/async_provisioning_device_client.py

```diff
@@ -17,11 +17,16 @@
 from azure.iot.device.provisioning.abstract_provisioning_device_client import (
     log_on_register_complete,
 )
-from azure.iot.device.provisioning.internal.polling_machine import PollingMachine
+from azure.iot.device.provisioning.pipeline import constant as dps_constant
 
 logger = logging.getLogger(__name__)
 
 
+async def handle_result(callback):
+    # TODO Define exceptions and handle
```

Comment:
># TODO Define exceptions and handle [](start = 4, length = 35)

This is probably going to look exactly like the code that Carter added to the iothub device clients.  Maybe that code can be moved into a common function used by all clients.